Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -1091,20 +1088,21 @@ void TerminalWidget::drop_event(GUI::DropEvent& event)
     if (event.mime_data().has_text()) {
         event.accept();
         auto text = event.mime_data().text();
-        write(m_ptm_fd, text.characters(), text.length());
+        send_non_user_input(text.bytes());
     } else if (event.mime_data().has_urls()) {
         event.accept();
         auto urls = event.mime_data().urls();
         bool first = true;
         for (auto& url : event.mime_data().urls()) {
             if (!first) {
-                write(m_ptm_fd, " ", 1);
+                constexpr StringView space = " ";
+                send_non_user_input(space.bytes());
                 first = false;
             }
             if (url.protocol() == "file")
-                write(m_ptm_fd, url.path().characters(), url.path().length());
+                send_non_user_input(url.path().to_byte_buffer());
```

Comment:
Could we use `url.path().bytes()` to avoid extra allocations?

Reply:
(Same below)

Reply:
We can't skip the allocation in `URL.to_string()`, right?

Reply:
Nope, `url.to_string()` will end up allocating a string regardless, but it's not nice to allocate more and then copy into it :P